src/integration-test/tests/citizen/claimantResponse/pages/claimant-choose-how-to-proceed.ts

```diff
@@ -0,0 +1,23 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  radioSettlement: 'input[id=optionsignSettlementAgreement]',
+  radioCcj: 'input[id=optionrequestCCJ]',
+  submit: 'input[id=saveAndContinue]'
+}
+
+export class ClaimantChooseHowToProceed {
+
+  chooseSettlement (): void {
+    I.checkOption(fields.radioSettlement)
+    I.click(fields.submit)
+  }
+
+  chooseCcj (): void {
+    I.checkOption(fields.radioCcj)
+    I.click(fields.submit)
```

Comment:
same

Reply:
`I.checkOption(fields.radioCcj)` rename radio-button to maybe `I.checkOption(fields.radioRequestCcj)`

Reply:
Done